src/Workleap.OpenApi.MSBuild/OasdiffManager.cs

```diff
@@ -59,9 +59,19 @@ public async Task RunOasdiffAsync(IReadOnlyCollection<string> openApiSpecFiles,
             this._loggerWrapper.LogMessage("- Specification file path: {0}", MessageImportance.High, baseSpecFile);
             this._loggerWrapper.LogMessage("- Specification generated from code path: {0} \n", MessageImportance.High, generatedSpecFilePath);
             
-            var result = await this._processWrapper.RunProcessAsync(oasdiffExecutePath, new[] { "diff", baseSpecFile, generatedSpecFilePath, "--exclude-elements", "description,examples,title,summary", "-f", "text", "-o" }, cancellationToken);
```

Comment:
Is removing `-f text` related to treat warnings as errors? What's the difference with before?

Reply:
It was hiding some diff.

With text it was saying "No controller diff, but other things have changed"
Without text it is saying "Version went from 3.0.0 to 2.0.0"